Appraisals

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+appraise 'rubocop-0.49.1' do
```

Comment:
I think we need to check 4 set of versions in the matrix/appraisal:
* lowest supported 0.x version (`= 0.49.1`)
* version where a breaking change was done (`= 0.85.0`) -- see #51
* highest supported 0.x version (`< 1.0`)
* highest supported 1.x version (`>= 1.0`)

Reply:
Eventually, there are 8 versions of rubocop.